Adhoc/Rotate-Array/solution.py

```diff
@@ -0,0 +1,19 @@
+class Solution:
+    def rotate(self, nums, k):
+        """
+        :type nums: List[int]
+        :type k: int
+        :rtype: void Do not return anything, modify nums in-place instead.
+        """
+        length=len(nums)
+        k=k%length
```

Comment:
`k = k % length`
Also add spaces in the reverse function below if possible 
`start = start + 1`
`end = end - 1` etc.